### ISSUE TITLE
Figure out Django Internationalization/Translation stuff

### ISSUE THREAD

**Issue by theandrewbriggs** [smartchicago/chicago-early-learning#523]
This is one of the major blockers for this milestone, getting the built-in Django translation stuff to work. New features in this milestone can't be merged in until we account for Spanish-language translations on some new site copy.

Translations needed:
- [x] "Use your current location" on homepage

**Comment by smarziano** [comment]
@theandrewbriggs keep adding needed translations as you go.

Translations:
- "Use your current location" is "Usar su ubicación actual"

**Comment by theandrewbriggs** [comment]
Here's a google spreadsheet to keep track of phrases that need to be translated and added to the Django internationalization file: https://docs.google.com/spreadsheets/d/1pwaE_W9mlBgi_7Ff7zKnPnj2uPv-kSr2bKsjOuCXg6E/edit?usp=sharing

**Comment by smarziano** [comment]
@theandrewbriggs I added translations to the spreadsheet. I thought we discussed changing the text of the search bar on the homepage to "Enter an address, center, or zip code" in English. I provided the Spanish translation for that as well.

**Comment by theandrewbriggs** [comment]
Ah yep I'll changed that.

**Comment by smarziano** [comment]
This still needs to be completed. Moving to "Phase 1" milestone. Still a high priority.

**Comment by smarziano** [comment]
At this time, the majority of translations have been added to staging. Therefore, adding "waiting on merge" label.

We will continue to use the spreadsheet to capture additional translations as they come up.

**Comment by theandrewbriggs** [comment]
Pushed to production. I assume there will be a need for new translations as we get new copy from the city; I vote closing this and opening up a separate issue as that arises.

**Comment by smarziano** [comment]
Agreed about closing this issue and opening others as they come up. Closing now!
